### PR TITLE
[13.0] sale_product_set: allow to pass custom lines to wizard

### PR DESCRIPTION
**Pull request by simahawk**
Includes some more improvements:

* views/actions: allow to add the set to an order from set view as well
* view: allow to ciew/select/unselect lines
* tests: cleanup test case and unify vars

### CODE REVIEW

File: sale_product_set/wizard/product_set_add.py [hunk 1]
```diff
@@ -14,13 +14,23 @@ class ProductSetAdd(models.TransientModel):
         "sale.order",
         "Sale Order",
         required=True,
-        default=lambda self: self.env.context.get("active_id"),
+        default=lambda self: self.env.context.get("active_id")
+        if self.env.context.get("active_model") == "sale.order"
+        else None,
```
Comment (mmequignon) [inline sale_product_set/wizard/product_set_add.py:19]: Isn't False the value to set for that?
```suggestion
        else False,
```

**Review by mmequignon — APPROVED** [review]
Only code review

**Review by TDu — APPROVED** [review]

File: sale_product_set/wizard/product_set_add.py (hunk 1 above)
Comment (simahawk) [reply sale_product_set/wizard/product_set_add.py:19]: it's the same

**Pull request merged**